app/models/submission/file.rb

```diff
@@ -3,7 +3,7 @@ class Submission::File < ApplicationRecord
 
   attr_writer :content
 
-  URI_REGEX = %r{s3://(?<bucket>[a-z0-9-]+)/(?<key>.*)}.freeze
+  URI_REGEX = %r{s3://(?<bucket>[a-z0-9-]+)/(?<key>.*)}
```

Comment:
Did this get unfrozen for a reason? Are all strings frozen by default now or something?

Reply:
I think this was rubocop. I'll double check. And does `%r` return a string or a regex object?

Reply:
It returns a Regular expression which is not mutable.

```irb
3.0.3 :001 > %r{\w+}.class
 => Regexp 
```

Reply:
Thanks.